### PR TITLE
[functionapp]: add ability to create java apps on Linux

### PR DESCRIPTION
**Pull request by ankitkumarr**
cc: @panchagnula @kulkarnisonia16 @amamounelsayed 

Fixes https://github.com/Azure/Azure-Functions/issues/1369, fixes https://github.com/Azure/azure-cli/issues/10905

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S164.

**Comment by ankitkumarr** [comment]
This should be good to merge. I don't have access to merge. Thanks for the approval!

**Comment by yonzhan** [comment]
@qwordy please help with the PR.

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -26,6 +26,7 @@ Release History
 * Fix issue #7154: Updating documentation for command <> to use back ticks instead of single quotes
 * Fix issue #11287: webapp up: By default make the app created using up 'should be 'SSL enabled'
 * Fix issue #11592: Add az webapp up flag for html static sites
+* functionapp: Add ability to create Java function apps in Linux
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst]: Please place your release note in the right position, which should be above 2.0.79

**Comment by Juliehzl** [comment]
@ankitkumarr may I know which api version is used for your feature?

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (ankitkumarr) [reply src/azure-cli/HISTORY.rst]: Thanks, @Juliehzl, just fixed.

**Comment by ankitkumarr** [comment]
@Juliehzl, there is no backend API change for this work. I only added changes in our client to modify the payload it sends.

**Comment by ankitkumarr** [comment]
@qwordy @Juliehzl , this should be good to merge unless you have any concerns.

**Comment by fabiocav** [comment]
@qwordy @Juliehzl ping on this. Can you please assist and have this merged?

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -6,6 +6,10 @@ Release History
 2.0.80
 ++++++
 
+**AppService**
+
+* functionapp: Add ability to create Java function apps in Linux
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst]: 2.0.80 has been released. Please move this entry above 2.0.80. Thanks.

**Comment by yonzhan** [comment]
@fabiocav  please address the comments raised by @qwordy before this build tomorrow.

**Comment by ankitkumarr** [comment]
@yonzhan @qwordy, I have updated it. Please merge it for this release, it should be good now.

If there's any minor History.rst update that may need in case, please feel free to amend my commit / update my PR and merge this. I am away at the moment and will not be available for the next couple of days.

**Pull request merged**